examples/api/lib/widgets/linked_text/linked_text.1.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+// This example demonstrates highlighting and linking Twitter handles.
```

Comment:
@craiglabenz Just to double check. Is it ok to use twitter as an example in this example?

Reply:
Good idea to check.  I could change it to username or something like that if needed.